Makefile

```diff
@@ -130,16 +130,13 @@ test:
 	@echo
 	go test -tags "${NOTARY_BUILDTAGS}" $(TESTOPTS) $(PKGS)
 
-test-full: TESTOPTS =
```

Comment:
yeah I'm fine with either removing it or bringing it in parity with circle.  Agree that the current version isn't particularly useful